app/presenters/organisation_content_presenter.rb

```diff
@@ -52,7 +52,7 @@ def expanded_links(content_id, keys, pattern_to_remove = nil)
   end
 
   def join_strings(response, keys, pattern_to_remove)
-    results = response.dig('expanded_links', keys[0])
+    results = response.fetch('expanded_links', {}).fetch(keys[0], {})
     values = results.map { |result| result.dig(*keys.drop(1)) }
```

Comment:
I guess we might have the same problem with the `dig` on the next line.  We could use `result.dig(*keys.drop(1)) || ''` to make sure we get a string, or `values = results.map { ... }.compact` to compress the results by throwing away any `nil` entries.

Reply:
Good idea